src/components/common/deleteResource.jsx

```diff
@@ -71,9 +72,13 @@ export class DeleteResourceModal extends React.Component {
                    titleIconVariant="warning"
                    footer={
                        <>
-                           <Button variant='danger' isLoading={this.state.inProgress} isDisabled={this.state.inProgress} onClick={this.delete}>
+                           <Button variant='danger' id="delete-resource-modal-primary" isLoading={this.state.inProgress} isDisabled={this.state.inProgress} onClick={this.delete}>
                                {actionName || _("Delete")}
                            </Button>
+                           {actionNameSecondary &&
+                           <Button variant='secondary' id="delete-resource-modal-secondary" isLoading={this.state.inProgress} isDisabled={this.state.inProgress} onClick={() => this.delete(false)} isDanger>
```

Comment:
I find this hard to read. Boolean arguments aren't nice. How about passing it the actual handler instead? I think `this.delete(this.props.deleteHandlerSecondary)` would be much cleaner?